erts/emulator/beam/arith_instrs.tab

```diff
@@ -320,7 +307,9 @@ shift.execute(Fail, Live, Dst) {
         }
```

Comment:
Should we move the entire big_shift logic out of the emulator and into big.c?

Reply:
I would want to consider that question separately from this pull request.